fastlane/Fastfile

```diff
@@ -32,6 +34,42 @@ platform :ios do
     increment_version_number(version_number: new_version_number, xcodeproj: "ios/PurchasesHybridCommon/PurchasesHybridCommon.xcodeproj")
     version_bump_podspec(path: "PurchasesHybridCommon.podspec", version_number: new_version_number)
   end
+  
+  desc "Release to CocoaPods, create Carthage archive, and create GitHub release"
+  lane :release do |options|
+    version_number = current_version_number
+    push_pods
+    carthage_archive
+    github_release(version: version_number)
```

Comment:
ideally we would refactor `:release` to do:
1. Release Android
2. Release iOS
3. After the previous have succeeded, make GitHub release

Now, the GitHub release is going to be done after iOS is released and not wait for Android to finish. I didn't want to refactor the Android deploys (they are currently performed directly in config.yml and not using fastlane), because the main focus of this PR was fixing iOS, which is currently failing if done manually with M1 computers.